src/main/resources/db/migration/V0_6__Update_fee_table.sql

```diff
@@ -0,0 +1,12 @@
+do
+$$
+    begin
+        if not exists(select from pg_type where typname = 'fee_status') then
+            create type "fee_status" as enum ('UNPAID', 'PAID', 'LATE');
+        end if;
+    end
+$$;
+
```

Comment:
Put this script inside an another script and be more explicit. Eg : add_fee_status.